A39-xds-http-filters.md

```diff
@@ -78,12 +78,14 @@ message type for its top-level configuration, but not all filters
 support override configuration at all, and those that do may or may not
 use the same message type for their override configuration as for their
 top-level configuration.  Therefore, every filter will be registered
-one or more message types.
+by one or more message types.
 
 Each filter implementation will provide the following:
 - Method(s) for validating the config protos when parsing an xDS response.
 - An indication of whether it is supported on the gRPC client and/or gRPC
   server.
+- An indication of whether the filter is a terminal filter (i.e., it
```

Comment:
It seems we could delete this and change the later text to read
> In addition,
 validation will fail if a terminal filter is not the last filter in
 the chain or if a non-terminal filter is the last filter in the chain.
 The only currently-supported terminal filter is
  `envoy.extensions.filters.http.router.v3.Router`.
 Any validation error will cause the Listener resource to be NACKed.

That allows you to use the registry if you want without implying that we support other terminal filters.

Reply:
Updated wording to clarify this.